public/app/plugins/datasource/grafana-azure-monitor-datasource/components/InsightsAnalyticsEditor/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Alert, CodeEditor, Select } from '@grafana/ui';
+import React from 'react';
+import { AzureMonitorOption, AzureMonitorQuery, AzureResultFormat } from '../../types';
+import { findOption } from '../../utils/common';
+import { Field } from '../Field';
+
+interface InsightsAnalyticsEditorProps {
+  query: AzureMonitorQuery;
+}
+
+const FORMAT_OPTIONS: Array<AzureMonitorOption<AzureResultFormat>> = [
+  { label: 'Time series', value: 'time_series' },
+  { label: 'Table', value: 'table' },
+];
+
+const InsightsAnalyticsEditor: React.FC<InsightsAnalyticsEditorProps> = ({ query }) => {
+  return (
+    <div data-testid="azure-monitor-insights-analytics-query-editor">
+      <CodeEditor
+        language="kusto"
+        value={query.insightsAnalytics.query}
+        height={200}
+        width="100%"
+        readOnly={true}
+        showMiniMap={false}
+      />
+
+      <Field label="Format as">
+        <Select
+          inputId="azure-monitor-logs-workspaces-field"
+          value={findOption(FORMAT_OPTIONS, query.insightsAnalytics.resultFormat)}
+          disabled={true}
+          options={FORMAT_OPTIONS}
+          onChange={() => {}}
```

Comment:
I guess onChange for this and for the code editor is not handled because this is all readonly now. If for some reason the resource centric logs pr won't be done for 8.0, will we remember to implement onChange?

Reply:
Yeah, correct. 

if resource-centric logs arent done for 8.0 we can just restore the angular view...

Reply:
:+1: